nixos/modules/services/web-apps/lotide.nix

```diff
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lotide;
+in
+{
+  options.services.lotide = {
+    enable = mkEnableOption "the lotide service";
+
+    package = mkOption {
+      default = pkgs.lotide;
+      defaultText = "pkgs.lotide";
+      type = types.package;
+      description = "lotide package to use.";
+    };
+
+    databaseUrl = mkOption {
+      default = null;
+      type = types.string;
+      description = "Credentials for database connection.";
```

Comment:
Please explain that this option should only ever be used if `hitide` is to connect to a local database without a password (socket authentication) because the password will be stored in the nix store and that is a bad idea. Please promote the user to provide an `EnvironmentFile` friendly file containing the password, like so:
```
$ cat /run/secrets/hitide
DATABASE_URL=postgresql://user:passwort@somehost:5432/databasename
```

Please use an example that does not include a password.